modules/local/sage/append/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: sage_append
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - bioconda::hmftools-sage=3.4.3
```

Comment:
container version is 3.4.4

Reply:
Thank you so much for catching this, updated in https://github.com/nf-core/oncoanalyser/pull/72